Makefile

```diff
@@ -324,7 +337,6 @@ kind-load: $(KIND) #EXHELP Loads the currently constructed images into the KIND
 	$(CONTAINER_RUNTIME) save $(CATD_IMG) | $(KIND) load image-archive /dev/stdin --name $(KIND_CLUSTER_NAME)
 
 .PHONY: kind-deploy
-kind-deploy: export MANIFEST := $(STANDARD_RELEASE_MANIFEST)
```

Comment:
Deleting this causes two issues:
1. You are redirecting into nothing on L331
2. The `envsubst` command doesn't have a value on L333